packages/components/src/components/install-guide/record-instructions/Ruby.vue

```diff
@@ -114,7 +114,7 @@ import VTestsPrompt from './TestsPrompt.vue';
 import VWebFrameworkPrompt from './WebFrameworkPrompt.vue';
 
 export default {
-  name: 'Ruby',
+  name: 'RubyRecordingInstructions',
```

Comment:
I don't think these names actually do anything. Was this causing an issue?

Reply:
These names can be used as tag names, so [the rule](https://eslint.vuejs.org/rules/multi-word-component-names.html) enforces multi-word names to avoid conflict with any future HTML elements.